src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -95,6 +105,17 @@ public static boolean start(final RootDoc rootDoc) throws IOException {
          return new HelpDoclet().startProcessDocs(rootDoc);
      }
 
+     private void validateSettingsDir() {
+         if (!settingsDir.exists()) {
+             logger.error(SETTINGS_DIR_OPTION + " : " + settingsDir.getPath() + " does not exist - will fallback on classpath: " + DEFAULT_SETTINGS_CLASSPATH);
+             settingsDirIsValid = false;
+         }
+         else if (!settingsDir.isDirectory()) {
+             logger.error(SETTINGS_DIR_OPTION + " : " + settingsDir.getPath() + " is not a directory - will fallback on classpath: " + DEFAULT_SETTINGS_CLASSPATH);
+             settingsDirIsValid = false;
+         }
+     }
+
```

Comment:
I changed the logic as suggested.  Now the user must specify either the settingsDir or to use the default.